ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -246,27 +246,31 @@ Click CheckBox Single Model Serving Platforms
     ${status}=  Get Checkbox State Of Single Modelserving platforms
     IF  "${status}"=="False"
         Select Checkbox   ${SINGLE_MODEL_SERVING_CHECK_BOX}
+        Save Changes In Cluster Settings
```

Comment:
I don't like this, the Save should happen outside of these keywords - what if I want to chain multiple changes before saving?

Reply:
+1